platform/broadcom/sonic-platform-modules-accton/as7326-56x/classes/thermalutil.py

```diff
@@ -123,11 +120,11 @@ def _get_thermal_val(self, thermal_num):
                 return 0  
             file_str = check_file.read()
             search_str="average current temperature is"
-            print "file_str.find=%s"%file_str.find(search_str)
+            #print "file_str.find=%s"%file_str.find(search_str)
             str_len = len(search_str)
             idx=file_str.find(search_str)
             if idx==-1:
-                print "bcm sdk is not ready ,retrun 0"
+                #print "bcm sdk is not ready ,retrun 0"
```

Comment:
Is it possible to either remove all of these commented-out print statements, or convert them into `logging.debug()` or `logging.verbose()` messages?